<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VMInstructionForm</class>
 <widget class="QWidget" name="VMInstructionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>72</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="instructionNumberLabel">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Line #1</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="InstructionComboBox">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="register1LineEdit">
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>38</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>0x01</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="register2LineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>38</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>0x02</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="register3LineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>38</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>0x03</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLineEdit" name="register4LineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>38</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>0x04</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLineEdit" name="register5LineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>38</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>0x</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLineEdit" name="register6LineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>38</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>0x</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLineEdit" name="register7LineEdit">
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>38</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>0x</string>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QLineEdit" name="register8LineEdit">
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>38</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>0x</string>
     </property>
    </widget>
   </item>
   <item row="0" column="11">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="13">
    <widget class="QPushButton" name="addReadOutputPushButton">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>106</red>
           <green>104</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="0" column="14">
    <widget class="QPushButton" name="AddFieldPushButton">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>69</red>
           <green>69</green>
           <blue>69</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>106</red>
           <green>104</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Add register fields.</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="15">
    <widget class="QPushButton" name="removeReadOutputPushButton">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>106</red>
           <green>104</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="0" column="16">
    <widget class="QPushButton" name="RemoveFieldPushButton">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>106</red>
           <green>104</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="toolTip">
      <string>Remove register fields.</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="0" column="12">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
